spec/schema.json

```diff
@@ -403,6 +409,18 @@
         }
       }
     },
+    "Ineligible": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "type": "string",
+          "enum": [
+            "INELIGIBLE_COMPANY_STRUCK_OFF",
+            "INELIGIBLE_NO_DISSOLUTION_ACTION"
+          ]
+        }
+      }
+    },
     "Status": {
       "type": "string",
       "enum": [
```

Comment:
We still need those two new statuses in the Status type, as per Mark's original comment.

Reply:
Reintroduced these.